Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -437,4 +461,22 @@ public void onUpdate(String entityId, Calendar lastSyncTime, boolean isSynced) {
             mPreferences.edit().putLong(entityId, lastSyncTime.getTimeInMillis()).apply();
         }
     }
+
+    private class VerificationListener implements AccountVerificationWatcher.VerificationStateListener {
+        @Override
+        public void onUpdate(AccountVerificationWatcher.Status status) {
+            switch (status) {
+                case VERIFIED:
+                    dismissReviewAccountDialog();
```

Comment:
are these wrapper functions necessary or could we inline their calls, given that this all happens from inside `Simplenote` now?